plugins/brand-openedx.yml

```diff
@@ -0,0 +1,7 @@
+name: brand-openedx
+version: 0.1.0
+patches:
+  mfe-dockerfile-post-npm-install: |
+    RUN npm install '@edx/frontend-component-header@git+https://git@github.com/abstract-tech/community-theme-header.git#consistent-deps' --registry=$NPM_REGISTRY
```

Comment:
If you decide to change the branch from `consistent-deps` to `community-theme` it should be updated here too